content/videos/more-p5/github-pages-hosting/index.json

```diff
@@ -3,6 +3,7 @@
   "description": "How to host your p5.js sketch with GitHub pages, updated for 2020!",
   "videoNumber": "Video number",
   "videoId": "ZneWjyn18e8",
+  "canonicalTrack": "git-and-github-for-poets",
```

Comment:
I think I would consider this one to be "P5 TIPS & TRICKS" actually!

Reply:
Done, I'll mark this as resolved